R/FSelectorRFE.R

```diff
@@ -96,21 +101,19 @@ FSelectorRFE = R6Class("FSelectorRFE",
             self$importance = importance_average(learners, feature_names)
           }
           # Eliminate the most unimportant features
-          states = as.list(!feature_names %in% names(
-            self$importance[1:archive$n_batch]))
+          states = as.list(feature_names %in% names(self$importance[seq(i)]))
```

Comment:
Can `seq(i)` go out of bounds?

Reply:
We assert that `pars$subset_sizes` is in decreasing order, ` < length(feature_names)` and `> 1`. So it should not go out of bounds.

Reply:
And `self$importance` is always a vector of length `feature_names`